backend/djangoindia/api/serializers/event.py

```diff
@@ -55,6 +55,21 @@ class EventRegistrationSerializer(serializers.Serializer):
     other_links = serializers.URLField(required=False, allow_blank=True)
     rsvp = serializers.BooleanField(default=False)
     include_in_attendee_list = serializers.BooleanField(default=False)
+    first_time_attendee = serializers.BooleanField(default= True)
+    attendee_type = serializers.ChoiceField(choices=EventRegistration.AttendeeType)
 
     def create(self, validated_data):
         return EventRegistration.objects.create(**validated_data)
+    
+class EventAttendeeSerializer(serializers.Serializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = EventRegistration
+        fields = [
+            'full_name', 'professional_status', 'organization',
+            'linkedin', 'github', 'twitter', 'first_time_attendee',
+            'attendee_type',
+    ]

```

Comment:
This is kindof correct but we specify field type of each field explicitly as you can see in other serializers due to performance issues in ModelSerializer.